management/clipper_manager.py

```diff
@@ -312,25 +312,19 @@ def get_app_info(self, name):
         Returns
         -------
         dict or None
-            Returns a dict with the application info if found. If the application
-            is not registered, None is returned.
+            Returns a dictionary with the specified application's info. This
+            will contain the attribute name-value pairs that were provided to
+            `register_application`. If no application with name `name` is
+            registered with Clipper, None is returned.
```

Comment:
This method does not return `None` if no application with name `name` is registered. Rather, invoking `get_app_info("nonexistent_name")` returns the following JSON dictionary:

```
{
  "name": "nonexistent_name"
}
```